ngraph/frontend/onnx/frontend/src/op/clip.cpp

```diff
@@ -30,30 +30,17 @@ OutputVector clip(const Node& node) {
 namespace set_11 {
 OutputVector clip(const Node& node) {
```

Comment:
According to the ONNX spec https://github.com/onnx/onnx/blob/master/docs/Operators.md#inputs-1---3 `min` is an input at index 1 and `max` at index 2. You're retrieving them in a different order here. Are you sure this still works like previously? Also, I think you should add a test that covers a case when `min` is optional in the model and `max` is explicitly passed in.

Reply:
The order is correct and aligned with the common clamp/clip implementation (https://docs.openvino.ai/latest/openvino_docs_ops_activation_Clamp_1.html).

The math formula is:
![image](https://user-images.githubusercontent.com/20084299/140469917-634ea473-48c2-4213-ac38-3a496b96c639.png)

There are several onnx clip test backend tests covering default inputs (all passed, [CI log](https://dev.azure.com/openvinoci/dldt/_build/results?buildId=195634&view=logs&j=8e6fb299-7deb-52f1-2f4a-df91230aba00&t=f812e137-0344-51cb-e8b0-0ab752070263&l=1443)), but I can add more.